### PR TITLE
Make downloading assets more robust

### PR DESCRIPTION
**Pull request by nre-ableton**
This PR adds some common variables that are used for network-related properties, fixes the retry/delay used when downloading the Jenkins WAR, and adds retry/delay used when downloading PIMT.

### CODE REVIEW

**Review by mst-ableton — APPROVED** [review]
nice, lgtm!

File: vars/main.yml [hunk 1]
```diff
@@ -12,6 +12,15 @@ jenkins_command: "/usr/bin/java {{ jenkins_java_args }} \
   --prefix={{ jenkins_prefix | quote }} \
   {{ jenkins_extra_args }}"
 
+# Delay (in seconds) to wait between retries when downloading Jenkins and related assets
+jenkins_download_delay: 3
```
Comment (ala-ableton) [inline vars/main.yml:16]: @nre-ableton In a677ebd65ed9c63d11609ff846aca68c0c8c9007, why was the delay changed from 5 to 3?

Comment (nre-ableton) [reply vars/main.yml:16]: Probably a copy/paste error. :sweat_smile: See https://github.com/Ableton/ansible-role-jenkins-jcasc/pull/252

Comment (ala-ableton) [reply vars/main.yml:16]: Thanks!

**Pull request merged**